cmd/compliance.go

```diff
@@ -97,9 +98,12 @@ func init() {
 	complianceCmd.Flags().BoolP("debug", "D", false, "debug logging")
 
 	// Output control
-	complianceCmd.Flags().BoolP("json", "j", false, "json format")
-	complianceCmd.Flags().BoolP("basic", "b", false, "basic format")
-	complianceCmd.Flags().BoolP("detailed", "d", false, "detailed format")
+	complianceCmd.Flags().BoolP("json", "j", false, "output in json format")
+	complianceCmd.Flags().BoolP("basic", "b", false, "output in basic format")
+	complianceCmd.Flags().BoolP("detailed", "d", false, "output in detailed format")
+	complianceCmd.Flags().BoolP("color", "l", false, "output in colorful")
```

Comment:
`color mode on`

Reply:
this should always default to off.

Reply:
Our motive is to make it off until and unless explicity it is turned on via `--color=true` or `-l`

Reply:
Do we need to make it `on` or `off`, by default ?

Reply:
off

Reply:
so, yeah that's what parameter (false) signifies to.